### PR TITLE
Ensure error description is not nil

### PR DESCRIPTION
**Pull request by ksuh90**

### CODE REVIEW

File: lib/xeroizer/http_response.rb [hunk 1]
```diff
@@ -133,7 +133,9 @@ def raise_error!
         when 400
           raise Xeroizer::BadResponse.new(description)
         when 401
-          raise OAuth::TokenExpired.new(description) if description.include?("TokenExpired")
+          if description && description.include?("TokenExpired")
```
Comment (CloCkWeRX) [inline lib/xeroizer/http_response.rb:136]: Is it better to use the safe operator? In instances where it is blank but a 401; it will silently fail.

**Review by CloCkWeRX — CHANGES_REQUESTED** [review]

File: lib/xeroizer/http_response.rb (hunk 1 above)
Comment (nnc) [reply lib/xeroizer/http_response.rb:136]: Yeah, silent failures are the worst. I'd rather get a `NoMethodError` in that case.

**Comment by CloCkWeRX** [comment]
Closing stale PR for now.

File: lib/xeroizer/http_response.rb (hunk 1 above)
Comment (ksuh90) [reply lib/xeroizer/http_response.rb:136]: it shouldn't fail if description is nil.

**Pull request closed without merging**